### PR TITLE
Provide the runtime class documentation

### PR DESCRIPTION
**Pull request by cynepco3hahue**
The documentation will explain what annotations you can use together with the performance profile runtime class.

Signed-off-by: Artyom Lukianov <alukiano@redhat.com>

### CODE REVIEW

**Comment by cynepco3hahue** [comment]
@MarSik FYI

File: docs/runtime-class.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Performance profile runtime class
+
+The performance-addon-operators for each profile creates the runtime class that you can use under the pod specification. The runtime class allows you to configure a set of annotations:
```
Comment (Tal-or) [inline docs/runtime-class.md:3]: Nit: I would rephrase that: ```The performance-addon-operators creates for each profile a runtime class that...```
